docs/06_how-to-guides/30_key-value-api/kv_table/20_how-to-create-indexes-kv-table.md

```diff
@@ -61,7 +61,7 @@ class [[eosio::contract]] smrtcontract : public contract {
 3. Pass the name of the property for which the index is defined as the second parameter.
 4. Call `init()` of the base class in the constructor of `address_table` type and pass the contract name as the first parameter and the primary index defined previously as the second parameter.
 
-Refer to the following possible implementation of a unique index on property `account_name` using macro `KV_NAMED_INDEX`:
+Refer to the following reference implementation of a unique index on property `account_name` using macro `KV_NAMED_INDEX`:
```

Comment:
in the cpp example ... when calling init :

    address_table(eosio::name contract_name) {
      init(contract_name, account_name);
    }

should "contract_name" be "smrtcontract" ??

Reply:
this looks like a declaration not calling ...????

Reply:
the ctor for the address_table struct is called when one instantiates an object of that type